src/api/store/graph.py

```diff
@@ -87,6 +87,55 @@ def graph_actions_completed(self, context: Context, args) -> (Graph, MassEnergiz
       return None, CustomMassenergizeError(e)
 
 
+  def graph_actions_completed_by_team(self, context: Context, args) -> (Graph, MassEnergizeAPIError):
+    try:
+
+      team_id = args.get('team_id', None)
+
+      if not team_id:
+        return None, CustomMassenergizeError("Missing team_id field")
+
+      team: Team = Team.objects.get(id=team_id)
+
+      if not team:
+        return None, InvalidResourceError()
+
+      if context.is_prod and not context.user_is_admin():
+        if not team.is_published:
+          return None, CustomMassenergizeError("Content Not Available Yet")
+  
+      members = TeamMember.objects.filter(team=team).all()
+
+      completed_action_rels = []
+      for member in members:
+        completed_action_rels.extend(member.user.useractionrel_set.filter(status="DONE").all())
+
+      categories = TagCollection.objects.get(name="Category").tag_set.all()
+
+      prefetch_related_objects(completed_action_rels, "action__tags")
+      data = [
+        {
+          "id"   : category.id,
+          "name" : category.name,
+          "value": len(list(filter(lambda action_rel : category in action_rel.action.tags.all(), completed_action_rels)))
+        }
+        for category in sorted(categories, key=lambda category: category.name)
+      ]
+
+      res = {
+        "data": data,
+        "title": "Actions Completed By Members Of Team",
+        "team": team.info()
+      }
+   
+      return res, None
+        
+    except Exception as e:
+      import traceback
```

Comment:
is this for your own debugging?
Do you want to ship this?

Reply:
I simply modelled this piece of code after what was done in the function above - `graph_actions_completed`, assumed it was desired

Reply:
yh i would like us to remove all those traces in favor of using a logging service.  for now you can keep it but we should be using a proper logging service for proper debugging.